### PR TITLE
Add exceptional test case

### PR DESCRIPTION
**Pull request by chrisdunelm**

### CODE REVIEW

**Comment by chrisdunelm** [comment]
There is an indentation error in the generated test in the PR. There is a TODO in the generation code to fix it.

File: tests/ProtoTests/Basic/Tests/BasicClientTest.php [hunk 1]
```diff
@@ -56,4 +57,35 @@ public function aMethodTest()
...
+        $transport->addResponse(null, $status);
+        // Mock request
+        try {
+            $client->aMethod();
+            $this->fail('Expected an ApiException, but no exception was thrown.');
+        } catch (ApiException $ex) {
+            $this->assertEquals($status->code, $ex->getCode());
+            $this->assertEquals($expectedExceptionMessage , $ex->getMessage());
+        }
+// Call popReceivedCalls to ensure the stub is exhausted
```
Comment (jskeet) [inline tests/ProtoTests/Basic/Tests/BasicClientTest.php:87]: How hard is it to fix this comment indentation?

**Review by jskeet — APPROVED** [review]
One niggle, but generally fine.

File: tests/ProtoTests/Basic/Tests/BasicClientTest.php (hunk 1 above)
Comment (chrisdunelm) [reply tests/ProtoTests/Basic/Tests/BasicClientTest.php:87]: Moderately hard :( As the code-formatter I'm currently using causes this.
I've got a TODO in the generation code to fix this.

**Pull request merged**